### PR TITLE
Add a brief description of features of NEST 3.0 to its index

### PR DESCRIPTION
**Pull request by jessica-mitchell**
Added some text to highlight the features beyond node and connection handling. It's meant to give a brief summary of what nest 3.0 offers, details are to be found in linked text.
There is a link for the stimulating backends, which doesnt exist yet (which will presumably exist once #1456 is merged)

### CODE REVIEW

**Comment by jougs** [comment]
@jessica-mitchell: I've updated your text a bit. Creating a branch somehow did not work, so I committed directly to your branch. Can you please have another look? Thanks!

File: doc/userdoc/guides/nest2_to_nest3/index.rst [hunk 1]
```diff
@@ -3,25 +3,28 @@ All about NEST 3.0
...
-In addition to the previously supported backends, SIONlib is now available.
-The interface for :doc:`stimulation devices <nest3_features/stimulation_backends>` can now handle data from an external source, such as another simulator.
-
-The Topology Module is no longer a separate module; it is integrated within ``nest``, now referred to as :doc:`spatially-structured networks <../spatial/index>`.
-
-:doc:`NEST Server <../../../nest_server>` provides a backend to run simulations that can be deployed locally or on a remote machine.
-The code that controls the simulation is sent over HTTP to NEST Server, thus you do not have to directly import ``nest``.
+NEST 3.0 introduces a more direct approach to accessing :doc:`neuron and device properties <nest3_features/nest3_node_handles>`
+and to :doc:`interacting with synapses <nest3_features/nest3_handling_connections>`. You can now read and write properties of
+nodes and connections with the functions ``get()`` and ``set()`` or by direct member access (e.g. ``v = neuron.V_m``).
```
Comment (jessica-mitchell) [inline doc/userdoc/guides/nest2_to_nest3/index.rst:8]:
```suggestion
nodes and connections with the functions ``get()`` and ``set()`` or by direct member access (e.g., ``v = neuron.V_m``). 
```

**Review by jessica-mitchell — COMMENTED** [review]
Looks good, just noticed a one missing comma in the e.g.

File: doc/userdoc/guides/nest2_to_nest3/index.rst (hunk 1 above)
Comment (jougs) [reply doc/userdoc/guides/nest2_to_nest3/index.rst:8]: Oh, those commas!

**Review by jougs — APPROVED** [review]

**Pull request merged**